test/mock/mock.jwks.service.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Whenever we choose to add something like this, I would like to a comment right next to it explaining why it's there. Otherwise it just looks like a magic hack to me.